openapi-specs/account.openapi.json

```diff
@@ -139,6 +139,63 @@
         ]
       }
     },
+    "/v1/delegation/revokeDelegation/{accountId}/{providerId}": {
+      "get": {
+        "operationId": "DelegationControllerV1_getRevokeDelegationPayload",
+        "summary": "Get a properly encoded RevokeDelegationPayload that can be signed",
+        "parameters": [
+          {
+            "name": "accountId",
+            "required": true,
+            "in": "path",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "providerId",
+            "required": true,
+            "in": "path",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returned an encoded RevokeDelegationPayload for signing"
+          }
```

Comment:
note this response type is empty; see previous comments about adding `type` annotations in the decorators.